### DISCUSSION TITLE
1 ESP Source w/ multiple others ESP sink

### DISCUSSION THREAD

**Discussion by isadorabit** [pschatzmann/ESP32-A2DP#375] [Q&A] [answered]
I want to use one EPS32 as Source and multiple others ESP32 as sink at the same time.
The goal is sending files from the first ESP32 to the others. All others will receive the same file.

Can i use this function to connect with multiple devices at once? How?  
     a2dp_source.start("MyESP1", get_data_channels);
 
* I want do the same for MyESP2, MyESP3 and MyESP4. 

(maybe some other function)

**Comment by pschatzmann** [comment, marked as answer]
No!
I suggest to use ESP-now to communicate between the devices. I suggest that you have a look at the example provided in the Arduino Audio Tools project
